hadoop/src/main/java/org/apache/carbondata/hadoop/CarbonInputFormat.java

```diff
@@ -768,29 +818,6 @@ public QueryModel getQueryModel(InputSplit inputSplit, TaskAttemptContext taskAt
     return queryModel;
   }
 
-  public CarbonReadSupport<T> getReadSupportClass(Configuration configuration) {
```

Comment:
no any code change, just move "set and get" method to together.  for example : put setFilterPredicates and getFilterPredicates to together.